sources/views/settings.js

```diff
@@ -2,16 +2,25 @@ import {JetView} from "webix-jet";
 
 export default class SettingsView extends JetView {
 	config() {
+		const _ = this.app.getService("locale")._;
 		return {
 			rows: [
 				{
-					view: "segmented", value: "RU", inputWidth: 250, options: [
-						{value: "RU"},
-						{value: "EN"}
-					], align: "center"
+					view: "segmented", localId: "language", value: "RU", inputWidth: 250, align: "center", options: [
```

Comment:
value:"RU" is incorrect, you use "en" and "ru" locales in your app
however, do not harcode this value as it depends on the active locale. get it as `langs.getLang()`

Reply:
still, the initial value defined as `langs.getLang()` is needed